app/views/patients/_abortion_information.html.erb

```diff
@@ -9,7 +9,7 @@
     <div class="row">
       <div class="col-sm-12 info-form">
         <h3> Clinic details </h3>
-        <%= bootstrap_form_for patient, html: { id: 'abortion-information-form' }, remote: true do |f| %>
+        <%= bootstrap_form_for patient, html: { id: 'abortion-information-form-1' }, remote: true do |f| %>
```

Comment:
This is necessary because, as this project has driven home, forms-within-forms lead to some seriously evil magic. This splits the main patient information form into two instead.